CONTRIBUTORS

```diff
@@ -0,0 +1,96 @@
+# This file lists the Contributors and Approvers for each SIG.
+# Permissions to GitHub repos are managed through teams in GitHub, which unfortunately are not
+# visible to everyone. For transparency, this file lists the members of the GitHub teams defined
+# for the ONNX organization.
+# 
+# To recommend someone be added as a Contributor or Approver, submit a PR to update this file
+# along with the justification. Once approved per the ONNX open governance guidelines, a 
+# Steering Committee admin will update the GitHub teams configuration and merge the PR.
+
+
+SIG-archinfra:
+    - zrphercule
+    - spandantiwari
+    - raymondxyang
+    - yuslepukhin
+    - faxu
+    - wschin
+    - kevinch-nv
+    - winnietsang
+    - chinhuang007
+    releasemgrs:
+    mlir:
```

Comment:
do we need these categories? if so, we should probably adjust the approvers category below.

Reply:
releasemgrs is a temporary group where whoever has volunteered to do the release is added - it gives them the permissions necessary to do the release activities. the volunteer doesn't have to be an approver.

mlir is for the mlir repo. there are a number of folks actively working on that so they have their own permissions. open to suggestions on how to unify.